tests/integ/scala/test_dataframe_writer_suite.py

```diff
@@ -139,6 +135,56 @@ def test_negative_write_with_target_column_name_order(session):
         session.table(table_name).drop_table()
 
 
+def test_write_with_target_column_name_order_all_kinds_of_dataframes_without_truncates(
```

Comment:
Let's add a test case when we try to insert with a subset of the columns where the missing columns are nullable in the table schema.

Reply:
ack.